migration/create-environment.sh

```diff
@@ -0,0 +1,237 @@
+#!/usr/bin/env bash
+set -ex
+
+ENVIRONMENT=$1
+
+#Logging to cft sbox to pull secrets
+az aks get-credentials --resource-group cft-sbox-00-rg --name cft-sbox-00-aks --subscription b72ab7b7-723f-4b18-b6f6-03b0f2c6a1bb --overwrite-existing -a && kubectl config use-context cft-sbox-00-aks-admin
```

Comment:
is the use-context needed? won't it swap you anyway?

Reply:
never realised, always had it stored that way :)